draft-ietf-quic-multipath.md

```diff
@@ -1346,7 +1346,11 @@ MAX_PATH_ID frames that do not increase the path limit MUST be ignored.
 
 A sender SHOULD send a PATHS_BLOCKED frame (type=0x15228c0d) when
 it wishes to open a path but is unable to do so due to the maximum path identifier
-limit set by its peer;
+limit set by its peer.
+Note that PATH_BLOCKED frame is informational. Sending a PATH_BLOCKED frame does not
```

Comment:
These should both say PATHS_BLOCKED rather than PATH_BLOCKED

Reply:
Nice catch! Updated.